### ISSUE TITLE
linter doesn't work with local plugins

### ISSUE THREAD

**Issue by joscha** [buildkite-plugins/buildkite-plugin-linter#372]
E.g. a readme with:
````md
```yml
steps:
  - command: test.sh
    plugins:
      - ./path/to/plugin: ~
```
````

and:
```yml
version: "3.4"
services:
  lint:
    image: buildkite/plugin-linter
    command: ['--id', './path/to/plugin']
    volumes:
      - ".:/plugin:ro"
```

does result in a:
```
  | ok 1 - plugin.yml is valid
  | not ok 2 - No readme config examples found with plugin id './path/to/plugin'. Have you configured the linter with the correct plugin id?
  | ok 3 - Readme version numbers are up-to-date (undefined)
  | 1..3
```

or am I holding it wrong?

**Comment by joscha** [comment]
ah, I think it is because of: https://github.com/buildkite-plugins/buildkite-plugin-linter/blob/e054233b36ecc1d518fbeae82f1f31bc13acea5a/lib/linters/example-linter.js#L80 - local plugins do not have a version and thus the regex won't match.

**Comment by joscha** [comment]
fix is in #373
